content/blog/2017-11-11-rperseus.md

```diff
@@ -0,0 +1,106 @@
+
+My best papers often sprung from the outer margins of my *Nestle-Aland Novum Testamentum Graece.* Here the editors inserted cross references to parallel vocabulary, themes, and even grammatical constructions. Given the intertextuality of biblical literature, the margins are a rich source of questions: Where else does the author use similar vocabulary? How is the source material used differently? Does the literary context affect our interpretation of this word? This is exploratory data analysis in biblical studies.     
+
+Unfortunately the excitement of the question is incommensurate with the tedium of the process--EDA continues by flipping back and forth between books, dog-earring pages, and avoiding paper cuts. `rperseus` aims to streamline this process with two functions: `get_perseus_text` and `perseus_parallel`. The former returns a data frame containing the text from any work in the Perseus Digital Library, and the latter renders a parallel in `ggplot2`. 
+
+Suppose I am writing a paper on different expressions of love in Paul’s letters. Naturally, I start in 1 Corinthians 13, the famed “Love Chapter” often heard at weddings and seen on bumper stickers. I finish the chapter and turn to the margins. In the image below, I see references to Colossians 1:4, 1 Thessalonians 1:3, 5:8, Hebrews 10:22-24, and Romans 8:35-39.
+
+![](/img/blog-images/2017-11-17-rperseus/nantg.png)
+*1 Corinithians 13 in Nestle-Aland Novum Testamentum Graece*
+
+Ignoring that some scholars exclude Colossians from the “authentic” letters, let’s see the references alongside each other:
+
+```r
+library(rperseus) #devtools::install_github(“ropensci/rperseus”)
+library(tidyverse)
+
+tribble(
+  ~label, ~excerpt,
+  "Colossians", "1.4",
+  "1 Thessalonians", "1.3",
+  "1 Thessalonians", "5.8",
+  "Romans", "8.35-8.39"
+  ) %>% 
+  left_join(perseus_catalog) %>%
+  filter(language == "eng") %>%
+  select(urn, excerpt) %>%
+  pmap_df(get_perseus_text) %>%
+  perseus_parallel(words_per_row = 4)
+```
+![](/img/blog-images/2017-11-17-rperseus/Parallel1.png)
+
+A brief explanation: First, I specify the labels and excerpts within a tibble. Second, I join the lazily loaded `perseus_catalog` onto the data frame. Third, I filter for the Greek[^2] and select the columns containing the arguments required for `get_perseus_text`. Fourth, I map over each urn and excerpt, returning another data frame. Finally, I pipe the output into `perseus_parallel`.
+
+[^2]: The Greek text from the Perseus Digital Library is from 1885 standards. The advancement of textual criticism in the 20th century led to a more stable text you would find in current editions of the Greek New Testament.  
+
+The key word shared by each passage is *agape* (“love”). Without going into details, it might be fruitful to consider the references alongside each other, perhaps pondering how the semantic range of *agape* expands or contracts within the Pauline corpus. 
```

Comment:
This phrase, "...pondering how the semantic range of *agape* expands or contracts within the Pauline corpus." had me wishing for a bit more explanation of why it might be fruitful to consider i.e. this is the one spot where it felt a bit jargony for our typical audience because most of us don't share your background.

Reply:
Added: 

> Paul had a penchant for appropriating and recasting old ideas--often in slippery and unexpected ways--and the dictionary is only an approximation. In other words, how can we move from the dictionary definition of *agape* towards Paul's unique vision?